osu.Game/IO/Legacy/SerializationReader.cs

```diff
@@ -241,7 +241,7 @@ public override Type BindToType(string assemblyName, string typeName)
                         {
                             if (typ.Contains("Version"))
                             {
-                                string asmTmp = typ.Substring(typ.IndexOf(',') + 1);
+                                string asmTmp = typ[(typ.IndexOf(',') + 1)..];
```

Comment:
jesus

Reply:
So shall we disable this inspection? Because now it's an error level thing.

Reply:
it was never enabled. your IDE is broken; fix your local settings.

Reply:
https://github.com/ppy/osu/blob/01961fe4f9739d161f35b8b7c69e2de631f96c3e/.editorconfig#L171
It was

Reply:
What is throwing an error from that?

Reply:
Visual Studio
![image](https://user-images.githubusercontent.com/50285552/124491163-8b584800-dde5-11eb-851e-557c49bb3538.png)

Reply:
Please remove it then, yes.